ISISComputingGroup/ibex_gui
# Ticket873: Added show/hide block to right clicking a group

### Description of work

Fixes https://github.com/ISISComputingGroup/IBEX/issues/873.

### To test

* Right click on group view but outside a group, confirm that show/hide blocks appears and works
* Right click on white space in a group, confirm that show/hide blocks appears and works
* Right click on a block, confirm that no new menu appears.
* Confirm that when blocks are shown/hidden the right click menu changes appropriately

---

#### Code Review


### Functional Tests


### Final Steps
